docs/src/wallet_to_wallet_with_tariscript.md

```diff
@@ -0,0 +1,205 @@
+# Wallet to Wallet transaction negotiation with TariScript
+
+This write up seeks to describe the detailed interaction between two wallets who are negotiating a Mimblewimble (MW) transaction 
+that includes a TariScript. The goal is to present the full set of operations that each party needs to perform, what data is 
+shared between the parties and also what data is retained by the receiver in order to spend the received UTXO in the future. 
+
+For a full breakdown of TariScript please see this [RFC](https://rfc.tari.com/RFC-0201_TariScript.html). This document will 
+repeat some of the detail contained in the RFC so that it is self-contained.
+
+We will use the same [notation](https://rfc.tari.com/RFC-0201_TariScript.html#notation) described in the RFC.
+
+## Alice sends a transaction to Bob
+Alice is going to send a transaction to Bob which spends a commitment \\( C_a \\), with a spending script \\( \alpha_a \\), 
+to a commitment \\( C_b \\), with a spending script \\( \alpha_b \\) that Bob will use to spend the new UTXO in the future.
+
+### Alice Round 1
+To spend \\( C_a \\) Alice must know the following:
+1. Spending key \\( k_a \\) of \\( C_a \\),
+2. Value \\( v_a \\) of \\( C_a \\),
+3. Features  \\( F_a \\) of \\( C_a \\),
+4. Script \\( \alpha_a \\) whose hash \\( \sigma_a = H(\alpha_a) \\),
+5. Script input \\( \theta_a \\) for script \\( \alpha_a \\),
+6. Script key \\( k_{sa} \\) whose public value (\\( K_{sa} = k_{sa} \cdot G \\)) is the key left over when script \\( \alpha_a \\) is executed with input \\( \theta_a \\)   
+7. Height \\( h_a \\) where \\( C_a \\) was mined.
+
+Alice then chooses the following values:
+1. Amount \\( v_b \\) being sent to Bob
+2. Fee value \\( f \\). The amounts need to balance out so \\( v_a = v_b + f \\)    
+3. A random nonce \\( r_a \\) whose public value is \\( R_a = r_a \cdot G \\)
+4. An UTXO offset key for Bob \\( k_{Ob} \\) whose public value is \\( K_{Ob} = k_{Ob} \cdot G \\)
```

Comment:
```suggestion
4. An UTXO offset key \\( k_{Ob} \\)  for Bob's UTXO \\( C_b \\)  whose public value is \\( K_{Ob} = k_{Ob} \cdot G \\)
```
In an Alice send to Bob use case, the offset key is chosen by Alice, and only Alice knows the offset key \\( k_{Ob} \\) 

EDIT, I see it is correct below, but the text here above confused me.

Reply:
Will update to be clearer.